### DISCUSSION TITLE
Failed to open output file

### DISCUSSION THREAD

**Discussion by Emmanuel-324** [idaholab/moose#28650] [Q&A General]
### Check these boxes if you have followed the posting rules.

- [X] Q&A General is the most appropriate section for my question
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [X] I have searched the MOOSE website and the documentation does not answer my question
- [X] I have formatted my post following the posting guidelines (screenshots as a last resort, triple back quotes around pasted text)

### Question

I am running an input script and it failed due to this error
```
*** ERROR ***
Failed to open fileTest_5_two_phase_test_trial_5_new_out.e-s008[39m
```

**Comment by GiudGiud** [comment]
Hello

we need more information here. What were you running, what systems were in use? etc

**Reply by Emmanuel-324** [reply]
I am running a phase field model using kks. Everything was working fine until all my current running jobs failed due to similar error messages. I am using an HPC cluster

**Reply by GiudGiud** [reply]
did you meet the storage limit on the HPC cluster? This would explain why the file cant be created/read
